compiler/src/iree/compiler/GlobalOptimization/GeneralizeLinalgNamedOps.cpp

```diff
@@ -62,7 +63,12 @@ void GeneralizeLinalgNamedOpsPass::runOnOperation() {
   auto funcOp = getOperation();
   SmallVector<linalg::LinalgOp> namedOpCandidates;
   funcOp.walk([&](linalg::LinalgOp linalgOp) {
-    if (!IREE::Flow::isNonNullAndOutsideDispatch(linalgOp)) {
+    if (!IREE::Flow::isNonNullAndOutsideDispatch(linalgOp) ||
```

Comment:
I dont think we need this here. This is in global optimizations, where there is no dispatches. So this seems unnecessary.

Reply:
We have such code in few GlobalOptimization passes. I thought that it is because people could pre-wrap dispatches? Is it still the case?